src/layouts/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './Header.module.css';
+
+interface HeaderProps {
+  title: string;
+  Icon: React.ReactElement;
+}
+
+function Header({ title, Icon }: HeaderProps) {
```

Comment:
The Header component is now a layout component, it will make more sense when implementing the filters. but It should contain all margins between components in order to keep the components composable.